introduction.md

```diff
@@ -1 +1,5 @@
 Hello world this is briandcase, my favorite path is electives
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?